analysis_options.yaml

```diff
@@ -0,0 +1,6 @@
+include: package:pedantic/analysis_options.yaml
+analyzer:
+  strong-mode:
+    implicit-casts: false
+  language:
+    strict-raw-types: true
```

Comment:
Is this a "thing" we're doing now?

Reply:
🤷‍♂ When it is easy it seems like a good idea.

`implicit-casts: false` will be at least partially enforced along with null safety. The difference is this way will also check casts from `dynamic` which upcoming Dart will continue to ignore, and checking that is still worth doing.

The `strict-raw-types` I'm less sure about the utility. I _think_ there will be a general trend in this direction, but I'd also be fine skipping it for now.